pysingfel/geometry/__init__.py

```diff
@@ -220,7 +220,7 @@ def assemble_image_stack_batch(image_stack, index_map):
 # The following functions are utilized to work on the particles
 ######################################################################
 
-def get_random_translations(atom_pos, beam_focus_size):
+def get_random_translations(atom_pos, beam_focus_size, jet_radius):
```

Comment:
Suggest not using __init__.py for defining your functions.This file has different purpose.

Reply:
The `geometry` module used to be a single humongous file, which I spent some time refactoring. I turned it into a (sub)package and I placed its functions into different files. The ones in `__init__` are simply the ones that I haven't moved yet.

Since @irischang020 is just modifying an already-existing function, I think she can just leave it where it is.
I'll move it out of `__init__` whenever I spend some more time cleaning up the code.